src/inference/include/openvino/runtime/auto/properties.hpp

```diff
@@ -74,5 +74,10 @@ inline std::istream& operator>>(std::istream& is, SchedulePolicy& policy) {
  * @ingroup ov_runtime_cpp_prop_api
  */
 static constexpr Property<SchedulePolicy> schedule_policy{"SCHEDULE_POLICY"};
+
+/**
+ * @brief setting the target device utilization threshold used for hw selection logic hw in AUTO
+ */
+static constexpr Property<std::map<std::string, double>> devices_utilization_threshold{"DEVICES_UTILIZATION_THRESHOLD"};
```

Comment:
Please add Python / C interface and test case for new property.

Reply:
updated.

Reply:
Yes, will update the value type to integral value.

Reply:
If reverted, then changes in ov::Any can also be reverted

Reply:
Do you mean that the code added in `any.hpp` should be remove from here?    
https://github.com/openvinotoolkit/openvino/blob/b0486f7d3805a78c60aafdbffac8775fe9858d19/src/core/include/openvino/core/any.hpp#L725